generators/generator-constants.js

```diff
@@ -18,7 +18,7 @@
  */
 
 // version of docker images
-const DOCKER_JHIPSTER_REGISTRY = 'jhipster/jhipster-registry:v3.2.4';
+const DOCKER_JHIPSTER_REGISTRY = 'jhipster/jhipster-registry:develop';
```

Comment:
Need to change this, as soon as the develop branch is merged into master and there is a new release of jhipster-registry